public/js/controllers/MarketController.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-app.controller('MarketController', ['ngDialog', 'ngTableParams', function($scope, carePriceService, $location, toaster) {
+app.controller('MarketController', ['ngDialog', 'ngTableParams', function($scope, carePriceService, $location, toaster, $routeParams) {
```

Comment:
* `$scope ` is not needed
* please update correctly each paramter in the string array matching each parameter inyected: `app.controller('MarketController', ['carePriceService', 'location', 'toaster','$routeParams', function( carePriceService, $location, toaster, $routeParams)`